.github/workflows/doc-lint-test.yaml

```diff
@@ -203,6 +206,54 @@ jobs:
           cache-from: type=registry,ref=${{ env.BUILDCACHE_IMAGE }}
           cache-to: type=registry,ref=ghcr.io/${{ env.BUILDCACHE_IMAGE }},mode=max
 
+#################################################
+# Release Note Added
+#################################################
+  check_new_release_note:
+    if: >
+      always() &&
+      needs.set_variables.outputs.CHECK_NEW_RELEASE_NOTE == 'true' &&
+      (needs.build_doclinttest_image.result == 'success' ||
+      needs.build_doclinttest_image.result == 'skipped')
```

Comment:
@jsolbrig was there a reason you had not put the release note check in this workflow before?  Or just didn't get around to it?

Reply:
I just didn't get around to it and it wasn't really needed because it ran just fine without needing to be in the docker image.